subsys/bluetooth/mesh/proxy.c

```diff
@@ -281,6 +282,12 @@ static void proxy_cfg(struct bt_mesh_proxy_client *client)
 		return;
 	}
 
+	if (is_replay(&rx, NULL)) {
+		BT_WARN("Replay: src 0x%04x dst 0x%04x seq 0x%06x",
+			rx.ctx.addr, rx.ctx.recv_dst, rx.seq);
+		return;
+	}
```

Comment:
The message should also be added to replay protection when it has been successfully processed.

Reply:
well, `is_replay` already store new RPL when it has been successfully processed.